src/lib/contracts/common/interfaces/IMembershipContract.ts

```diff
@@ -21,4 +21,6 @@ export interface IMembershipContract extends BaseContract {
     [bigint],
     "view"
   >;
+
+  nonces?: TypedContractMethod<[owner: AddressLike], [bigint], "view">;
```

Comment:
I added nonces as an optional method here just to prevent TypeScript from complaining. I have no idea if ERC1155 tokens implement EIP-2612. As of right now, this should never get called.